### PR TITLE
TCM-158 | Forms move top image into scrollview

### PR DESCRIPTION
**Pull request by halilyesilyurt**
### Summary

- [x] Moved email and login layouts into scrollview.
- [x] A few padding fixes.

### Issue

[TCM-158](https://hyfnla.atlassian.net/browse/TCM-158)

### Video/Screenshot

Big Screen

https://user-images.githubusercontent.com/30659854/114897906-5b1e9000-9e1a-11eb-9fcd-3b2e888db72c.mp4

Small Screen

https://user-images.githubusercontent.com/30659854/114897915-5d80ea00-9e1a-11eb-8b58-f2396a33799a.mp4

### CODE REVIEW

File: app/src/main/res/layout/fragment_email.xml [hunk 1]
```diff
@@ -1,133 +1,144 @@
...
-        android:layout_marginStart="16dp"
-        android:layout_marginBottom="16dp"
-        android:textColor="@color/white"
-        android:fontFamily="@font/atcarquette_bold"
-        android:textSize="32sp"
-        app:layout_constraintBottom_toBottomOf="@+id/ivHeader"
-        app:layout_constraintStart_toStartOf="parent"
-        tools:text="@string/email_forgot_title" />
+    android:fillViewport="true"
+    app:layout_constraintBottom_toBottomOf="parent"
```
Comment (VivekV95) [inline app/src/main/res/layout/fragment_email.xml:8]: Don't think the constraints are needed as we're matching parent dimensions

File: app/src/main/res/layout/fragment_login.xml [hunk 2]
```diff
@@ -1,149 +1,160 @@
...
-        android:layout_marginStart="16dp"
-        android:layout_marginBottom="16dp"
-        android:text="@string/login_log_in"
-        android:textColor="@color/white"
-        android:textSize="32sp"
-        android:textStyle="bold"
-        app:layout_constraintBottom_toBottomOf="@+id/ivHeader"
-        app:layout_constraintStart_toStartOf="parent" />
+    android:fillViewport="true"
+    app:layout_constraintBottom_toBottomOf="parent"
```
Comment (VivekV95) [inline app/src/main/res/layout/fragment_login.xml:7]: Same as email, don't think the constraints are needed

**Review by VivekV95 — CHANGES_REQUESTED** [review]
Couple small things, also while we're here, let's make the dividers `greyLight` instead of `grey` in `login`, `email`, and `register` like they are in iOS

File: app/src/main/res/layout/fragment_email.xml (hunk 1 above)
Comment (halilyesilyurt) [reply app/src/main/res/layout/fragment_email.xml]: Completed.

File: app/src/main/res/layout/fragment_login.xml (hunk 2 above)
Comment (halilyesilyurt) [reply app/src/main/res/layout/fragment_login.xml]: Completed.

**Comment by halilyesilyurt** [comment]
Added demos.

File: app/src/main/res/layout/fragment_email.xml [hunk 3]
```diff
@@ -6,128 +6,144 @@
...
+                android:id="@+id/ivEmail"
+                android:layout_width="0dp"
+                android:layout_height="60dp"
+                app:inputViewState="email"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/ivHeader" />
+
+            <View
+                android:id="@+id/vLine"
```
Comment (VivekV95) [inline app/src/main/res/layout/fragment_email.xml:76]: Let's make the dividers in `login`, `register`, and `email` `greyLight`

**Review by VivekV95 — CHANGES_REQUESTED** [review]
One last thing

File: app/src/main/res/layout/fragment_email.xml (hunk 3 above)
Comment (halilyesilyurt) [reply app/src/main/res/layout/fragment_email.xml:76]: Completed.

**Review by VivekV95 — APPROVED** [review]

**Review by onodude — APPROVED** [review]

**Pull request merged**